### PR TITLE
@eessex => RSS updates for Flipboard integration

### PR DESCRIPTION
**Pull request by kanaabe**
Refactors the router to use positronql instead -- no more setTimeout as a result!

Addresses https://artsyproduct.atlassian.net/browse/GROW-516

### CODE REVIEW

File: src/desktop/models/article.coffee [hunk 1]
```diff
@@ -148,7 +149,10 @@ module.exports = class Article extends Backbone.Model
     ''
 
   hasContributingAuthors: ->
-    @get('contributing_authors').length > 0
+    @get('contributing_authors')?.length > 0
+
+  hasAuthors: ->
+    @get('authors')?.length > 0
```
Comment (eessex) [inline src/desktop/models/article.coffee]: 👍

File: src/desktop/apps/rss/templates/news.jade [hunk 2]
```diff
@@ -13,6 +13,12 @@ rss(version='2.0', xmlns:atom='http://www.w3.org/2005/Atom' xmlns:content='http:
         title= article.get('thumbnail_title')
         link #{sd.APP_URL}/article/#{article.get('slug')}
         description= article.get('description')
+        author=article.byline() || "Artsy Editors"
+        if article.get('vertical')
+          category=article.get('vertical').name
+        if article.get('layout') === 'video'
+          - media = article.get('media')
+          enclosure(url=media.url type="video/mp4")
         content:encoded
```
Comment (eessex) [inline src/desktop/apps/rss/templates/news.jade]: The video is also rendered in the `articles.jade` file, or is this some other kind of meta?

Comment (kanaabe) [reply src/desktop/apps/rss/templates/news.jade]: Yep, this is a field that exists outside of the article content! But this reminds me that we should use thumbnail images for regular articles as the enclosure.

**Comment by kanaabe** [comment]
Ok to merge @eessex ?

**Pull request merged**